### PR TITLE
Add a lil' server-side reportback gallery rendering.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This is more for demoing than anything else, but it's nice to see everything working! The controller now fetches reportbacks for the given campaign, and each block renders a few of them into the feed. Wheeeeee! 🚀

screen shot 2017-02-10 at 10 45 18 am

### CODE REVIEW

File: resources/assets/components/block.scss [hunk 1]
```diff
@@ -21,4 +21,13 @@
...
   box-shadow: $box-shadow;
   border-radius: $sm-border-radius;
+  overflow: hidden;
+}
+
+.reportback-block {
+  padding: 0;
+}
+
+.padded {
```
Comment (DFurnes) [inline resources/assets/components/block.scss]: I'll pull this out into it's own file in another PR.

File: resources/views/blocks/reportbacks.blade.php [hunk 2]
```diff
@@ -1,3 +1,12 @@
-<div class="block placeholder">
-    Reportbacks
-</div>
+@for($i = 0; $i < data_get($block, 'additionalContent.count', 4); $i++)
+    @php($reportback = $reportbacks->pop())
```
Comment (DFurnes) [inline resources/views/blocks/reportbacks.blade.php]: This is probably a lil' hacky, but sooooo freakin' cool. 🌵

File: resources/views/blocks/reportbacks.blade.php [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+    @if($reportback)
+        <div class="block reportback-block">
+            <img src="{{ $reportback['media']['uri'] }}">
+            <div class="padded">
+                <h4>{{ app(\Faker\Generator::class)->firstName }}</h4>
+                <p class="footnote">{{ app(\Faker\Generator::class)->numberBetween(5, 100) }} jeans</p>
```
Comment (DFurnes) [inline resources/views/blocks/reportbacks.blade.php]: Another little placeholder that I'll refactor in a later PR.

File: resources/views/blocks/reportbacks.blade.php (hunk 2 above)
Comment (weerd) [inline resources/views/blocks/reportbacks.blade.php]: wtf `@php`?! when did that happen?

**Comment by DFurnes** [comment]
[As a dev, I want to have reportback data available in the browser and display it in each block.](https://trello.com/c/e8BCMXNI/266-as-a-dev-i-want-to-have-reportback-data-available-in-the-browser-and-display-it-in-each-block)

**Pull request merged**